utils/_context/_scenarios/__init__.py

```diff
@@ -471,7 +471,16 @@ def all_endtoend_scenarios(test_object):
     )
 
     tracing_config_nondefault = EndToEndScenario(
-        "TRACING_CONFIG_NONDEFAULT", weblog_env={"DD_TRACE_HTTP_SERVER_ERROR_STATUSES": "200-201,202"}, doc="",
+        "TRACING_CONFIG_NONDEFAULT",
+        weblog_env={
+            "DD_TRACE_HTTP_SERVER_ERROR_STATUSES": "200-201,202",
+            "DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP": "ssn=\d{3}-\d{2}-\d{4}",
+        },
+        doc="",
+    )
+
+    tracing_config_empty = EndToEndScenario(
+        "TRACING_CONFIG_EMPTY", weblog_env={"DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP": ""}, doc="",
```

Comment:
Could you rename it to `TRACING_CONFIG_NONDEFAULT_2`, otherwise, we'll have one scenario per non default value.